caravel/utils.py

```diff
@@ -212,6 +212,32 @@ def process_result_value(self, value, dialect):
 
 def init(caravel):
     """Inits the Caravel application with security roles and such"""
+    ADMIN_ONLY_VIEW_MENUES = set([
+        'ResetPasswordView',
+        'RoleModelView',
+        'Security',
+        'UserDBModelView',
+        'SQL Lab <span class="label label-danger">alpha</span>',
+        'AccessRequestsModelView',
```

Comment:
We probably need a new role for people who can grant rights to others that are not admin. It doesn't have to be part of this PR, your call.